### ISSUE TITLE
If the text contains '<', the following text will be recognized as beginning of another tag

### ISSUE THREAD

**Issue by AsukaSun** [cheeriojs/cheerio#929]
ignore the &:

<&td width="16%">75&<&abc≤xyz</&td>
<&td class="tdcenter" width="7%">mn</&td>

I will get a node named 'abc≤xyz<' and the next td will be its child

**Comment by shivasurya** [comment]
@AsukaSun  it's better to escape 75&<&abc≤xyz and try it. after parsing you can replace it whatever you want. i hope that will be a quicker fix.

**Comment by raldred** [comment]
+1 I'm generating html emails using foundation that contains ERB tags (which are a ruby tempting)
The tool chain uses cheerio for magnification and style inlining

I have the following:
`<div><%= some_ruby_variable %></div>`

Which is transformed into:
`<%= some_ruby_variable="" %=""></%=>`

It would be nice to have the ability to ignore certain things from parsing that look like tags.

**Comment by DianeLooney** [comment]
I know there's a transition to a new parser happening, so I'm not sure how long this will be relevant. That said, if you need a workaround then you can:

manually wrap the content you want excluded with `<? ... ?>` before it is loaded into cheerio
just before render, convert those elements to text and remove the ?'s from the data value
render with decodeEntities: false

```js
var cheerio = require("cheerio");
var data = `<table><tr><&td width="16%"><?75&<&abc≤xyz?></&td><&td class="tdcenter" width="7%">mn</&td></tr></table>`;
var $ = cheerio.load(data);
$("*").each((i, elt) => {
  elt.children.filter(x => x.type === "directive").forEach(val => {
    val.type = "text";
    val.data = val.data.slice(1, -1); //
  });
});
console.log($.html({ decodeEntities: false }));
```

This will remove any `<? ?>` tags you already have in your document. If you need those preserved, then you'll need to change the `$("*")` selector to ignore the parents of those tags.

There's probably a better solution, I'm not sure what it is.

**Comment by fb55** [comment]
Fixed by #985
